lib/streamlit/caching.py

```diff
@@ -387,10 +387,11 @@ def cache(
         the cached function.
 
     hash_funcs : dict or None
```

Comment:
dict or str or None

Reply:
`hash_funcs` is still `dict or None` but the `key` for the dict is now `type or str` but we don't specify the types for the key of the dict.